data/conf/iceberg/hive-site.xml

```diff
@@ -0,0 +1,321 @@
+<?xml version="1.0"?>
```

Comment:
do we need our own hive-site at this stage? Can not we get away with a general one?

Reply:
I needed a new conf, because of the `hive.in.iceberg.test=true` param.

Reply:
I would rather get rid of the `hive.in.iceberg.test` altogether

Reply:
Agreed that @lcspinter will discuss with @deniskuzZ the effort involved in removing the problematic part from `TxnHandler`. Depending on the time estimate though, we might want to temporarily live with this workaround, because this patch is blocking several other patches of Laci.

Reply:
I've discussed this with @deniskuzZ, and he advised that I should open a separate PR to remove this check around locking and update the unit tests which are expecting this exception.

Reply:
Sounds good! So that we don't forget, let's create a follow up Jira to remove the new iceberg-test flag once that locking PR has gone in.